glyphs2fontir/src/source.rs

```diff
@@ -845,7 +845,7 @@ impl Work<Context, WorkId, WorkError> for GlyphIrWork {
         }
 
         // https://github.com/googlefonts/fontmake-rs/issues/285 glyphs non-spacing marks are 0-width
```

Comment:
I think you should move the comment to the `if !is_nonspacing` if you rename it so